.github/workflows/integration-test-metrics.yml

```diff
@@ -0,0 +1,58 @@
+name: Integration-Test-Metrics
+on:
+  workflow_call:
+    inputs:
+      runtime_tag:
+        description: "Tag for the deno and python runner images"
+        type: "string"
+        required: true
+      cert-manager-io-enabled:
+        description: "Decides whether to use cert-manager.io"
+        type: "string"
+        default: "cert_manager_io_off"
+
+# Declare default permissions as read only.
+permissions: read-all
+
+env:
+  GO_VERSION: "~1.21"
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  run-integration-test-metrics:
+    name: Run Chainsaw Tests
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Check out code
+        uses: actions/checkout@b4ffde65f46336ab88eb53be808477a3936bae11 # v4.1.1
+
+      - name: Setup cluster
+        uses: ./.github/actions/deploy-keptn-on-cluster
+        with:
+          runtime_tag: ${{ inputs.runtime_tag }}
+          values_file: values-integration-metrics-${{ inputs.cert-manager-io-enabled }}.yaml
+          cert-manager-io-enabled: ${{ inputs.cert-manager-io-enabled }}
```

Comment:
do we still need this parameter?

Reply:
I guess we can leave it, it was testing the proper functionality of metrics-operator with cert-manager.io or keptn-cert-manager, but this is tested already by the global integration tests, I will remove the parameter.